zdm/craco/MC_Surveys/CRACO_alpha1_Planck18_Gamma.dat

```diff
@@ -0,0 +1,5015 @@
+BW    288 #MHz
+FRES  1 #MHz
+NFRB 1 #Number of FRBs
+DIAM 12
+NBEAMS 36
+BEAM lat50_log #prefix of beam file
+FBAR 1320
+TRES  1.7 #ms
+SNRTHRESH 9.5
+NFRB 100
```

Comment:
Make it 5000? Same for NORM_FRB

Reply:
fixed.  and for the future too (I hope).